ghedesigner/enums.py

```diff
@@ -16,6 +16,7 @@ class DoubleUTubeConnType(Enum):
 class TimestepType(Enum):
     HOURLY = auto()
     HYBRID = auto()
+    DYNAMIC = auto()
```

Comment:
You're proposing a new load aggregation scheme, not a timestep method, right? Unless I'm missing something, this should not be included here. Or maybe we also need some other cleanups in general to clarify the usage. Ideally, we should be able to have hourly timesteps and still use any of the load aggregation schemes that have been proposed.

Reply:
Thanks! Removed it, and included load aggregation to the Hourly timestep function.